src/Text/Pretty/Simple/Internal/Color.hs

```diff
@@ -39,15 +37,15 @@ import System.Console.ANSI
 --
 -- If you don't want to use a color for one of the options, use 'colorNull'.
 data ColorOptions = ColorOptions
-  { colorQuote :: Builder
+  { colorQuote :: AnsiStyle
```

Comment:
Colours are now expressed with [AnsiStyle](https://hackage.haskell.org/package/prettyprinter-ansi-terminal-1.1.1.2/docs/Data-Text-Prettyprint-Doc-Render-Terminal.html#t:AnsiStyle). This should actually be easier for users to work with, but is obviously a breaking change.